src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -141,14 +141,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String messageContent = request.getParameter("message");
 
     // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
+    // String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.basic());
 
     Message message =
         new Message(
             UUID.randomUUID(),
             conversation.getId(),
             user.getId(),
-            cleanedMessageContent,
+            messageContent,
```

Comment:
It looks like if you add a file larger than 1500 bytes, you get an error, but the image is still written. Can we have tests to deal with that (or change images to use a text field rather than the string field?)